ui/apps/platform/src/Containers/VulnMgmt/widgets/CvesByCvssScore.js

```diff
@@ -62,7 +59,12 @@ const CLUSTER_CVES_QUERY = gql`
     }
 `;
 
-const vulnerabilitySeveritySuffix = '_VULNERABILITY_SEVERITY';
+const vulnerabilitySeverities = [
+    'LOW_VULNERABILITY_SEVERITY',
+    'MODERATE_VULNERABILITY_SEVERITY',
+    'IMPORTANT_VULNERABILITY_SEVERITY',
+    'CRITICAL_VULNERABILITY_SEVERITY',
+];
```

Comment:
I think the PR yesterday also defined these for a different widget.

Not high-priority to refactor for VM 1.0, but we should start afresh with constants defined once in VM 2.0.

cc: @alwayshooin @dvail

Reply:
It was mapping from response data key to vulnerability severity constant:

```js
const vulnKeyMap = {
    low: 'LOW_VULNERABILITY_SEVERITY',
    moderate: 'MODERATE_VULNERABILITY_SEVERITY',
    important: 'IMPORTANT_VULNERABILITY_SEVERITY',
    critical: 'CRITICAL_VULNERABILITY_SEVERITY',
};
```

Reply:
In the following pull request, 2 components will each have a similar array of policy severities.

The reason for the duplication is because the order from low to critical, or vice versa, is component dependent. For example, Sunburst renders critical to low, even though its data prop is from low to high.